spec/models/disclosure_check_spec.rb

```diff
@@ -4,4 +4,22 @@
   subject { described_class.new(attributes) }
 
   let(:attributes) { {} }
+
+  describe '#relevant_order?' do
+    context 'when conviction_subtype is nil' do
+      it { expect { subject.relevant_order? }.to raise_error(NoMethodError) }
```

Comment:
I could `return false if conviction_subtype.nil?` in the method definition, but I thought that if one was not given something is definitely wrong somewhere and would be best to break so that we are notified about it.